.github/stale.yml

```
@@ -2,6 +2,10 @@ daysUntilStale: 60
 daysUntilClose: 7
 exemptLabels:
   - "status: accepted"
+  - "type: bug"
+  - "status: blocked"
+  - "future"
+  - "we must go deeper"
```

```suggestion
  - "we must go deeper"
  - "status: rebase required"
  - "status: in progress"
```
Adding rebase required & in progress, since it makes sense to not mark those as stale